cmd/p2p/query/query.go

```diff
@@ -0,0 +1,112 @@
+package query
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/eth/protocols/eth"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/maticnetwork/polygon-cli/p2p"
+)
+
+type (
+	queryParams struct {
+		StartBlock uint64
+		Amount     uint64
+	}
+)
+
+var (
+	inputQueryParams queryParams
+)
+
+var QueryCmd = &cobra.Command{
+	Use:   "query [enode/enr]",
+	Short: "Query block header(s) from node and prints the output.",
+	Long: `Query header of single block or range of blocks given a single enode/enr.
+	
+This command will initially establish a handshake and exchange status message
+from the peer. Then, it will query the node for block(s) given the start block
+and the amount of blocks to query and print the results.`,
+	Args: cobra.MinimumNArgs(1),
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if inputQueryParams.Amount < 1 {
+			return fmt.Errorf("amount must be greater than 0")
+		}
```

Comment:
Is this required? I think `inputQueryParams.Amount` has a default value of `1` (set by the `init()` method).